pkg/resource/tracing-secret/tracing-secret.go

```diff
@@ -0,0 +1,64 @@
+package tracingsecret
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	capi "sigs.k8s.io/cluster-api/api/v1beta1"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggingcredentials "github.com/giantswarm/logging-operator/pkg/resource/logging-credentials"
+)
+
+const (
+	TracingSecretName = "events-logger-secret" // #nosec G101
+)
+
+func generateTracingSecret(cluster *capi.Cluster) (v1.Secret, error) {
+	clusterName := cluster.GetName()
```

Comment:
Okay I think the issue might be coming from here. You do not need this reconciler because the tracing username and password should be added to the logging-secret/alloy-logging-secret.go file instead

Reply:
What you need to add is this https://github.com/giantswarm/logging-operator/tree/a873e893bff51051ad53f07643dfe5d9800b0ecc/pkg/resource/loki-ingress-auth-secret.

Technically, I think it might make more sense to do it this way https://github.com/giantswarm/observability-operator/pull/22/files#diff-d57b38ebd1c960ec16b89162cc722927634519acfc1dd8ba75fe003f1bc00328 and create a secret that contains all username: passwords  like this https://github.com/giantswarm/shared-configs/blob/e59b506b7c8cea780cb6bfda8a6473f0a0920b20/default/apps/mimir/configmap-values.yaml.template#L76

Reply:
We might want to use nginx.ingress.kubernetes.io/auth-secret-type: auth-map but we're not there yet